### PR TITLE
Make portable builds the default and remove non-portable official builds

### PR DESCRIPTION
**Pull request by weshaggard**
@gkhanna79 @MattGal @ericstj PTAL

This should flip corefx to have Portable builds by default and removes all the non-portable build legs in official builds.

### CODE REVIEW

**Comment by weshaggard** [comment]
I'm doing a private build of this branch to test out my changes.

**Comment by weshaggard** [comment]
I don't want to merge and go on holiday so I'm going to hold this until Tuesday when I'm back.

File: config.json [hunk 1]
```diff
@@ -411,7 +411,7 @@
            "CmakeBuildType": "default",
            "HostOs": "default",
            "ProcessorCount": "default",
-           "AdditionalArgs": "default",
```
Comment (gkhanna79) [inline config.json]: Have you confirmed that build.{sh|cmd} are passing the flag correctly all the way to the native component build?

Comment (weshaggard) [inline config.json]: Yes I did verify build.cmd and I will look at build.sh from the CI legs.

Comment (weshaggard) [inline config.json]: I verified

**Comment by weshaggard** [comment]
CI was green and the latest push only had changes to the official build pipeline so I'm merging and going to kick off an official build and monitor it.

**Pull request merged**